.github/workflows/pull_request_check_action.yml

```diff
@@ -48,10 +48,8 @@ jobs:
 
       - name: Setup the database before running migrations
         run: |
-          apt-get update
-          apt-get install -y --no-install-recommends postgresql-client
-          createdb $POSTGRES_TEST_DB
-          echo "CREATE EXTENSION postgis" | psql -d "postgresql://$POSTGRES_USER$POSTGRES_PASSWORD:@localhost:5432/$POSTGRES_TEST_DB"
+          sudo apt-get install postgresql-client
```

Comment:
Don't forget `-y` flag with `apt-get install` to autoconfirm all actions, otherwise, the command gets stuck.

Reply:
Okay, got it.